src/components/header/Search.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="w-full flex items-center px-5 sm:px-10">
-    <button @click="$store.dispatch('ui/setHeaderType', null)">
+    <button @click="$store.dispatch('ui/setHeaderType', null)" class="flex-none">
```

Comment:
```suggestion
    <button @click="$store.dispatch('ui/setHeaderType', null)" class="flex flex-no-shrink">
```

I'd do it like this, what do you think?

Reply:
I don't think you need to apply flex class again. 
And flex-none and flex-no-shrink imply the same thing here. flex-none restricts grow also. However, here svg won't grow though, so they both mean same

Reply:
can't test right now, but i think without `flex` the icon won't be vertically centered.

Reply:
Yup. you are right. Visually that doesn't make much difference. But yeah it'll be better.